assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -67,15 +68,24 @@ const PanelTitle = styled.span`
     error ? theme.colors.fg.negative : theme.colors.fg.warning};
 `;
 
-const Row = styled.div`
+const Row = styled.div.attrs({ role: 'listitem' })`
```

Comment:
How does this affect the accessibility tree? I don't see a parent with role `list`.

Reply:
I should add that, thanks.